resources/views/docs/info.md.blade.php

```diff
@@ -18,6 +18,12 @@
 
 Current rate limit is set at an insanely high 1200 requests per minute, with burst capability of up to 200 beyond that. If you require more, you probably fall into the above category of abuse. If you are doing more than 60 requests a minute, you should probably give [peppy](mailto:pe@ppy.sh) a yell.
 
+# Wrappers
+
+There are community maintained wrappers for the api:
```

Comment:
```suggestion
Below is a list of some language-specific wrappers maintained by the community. Your mileage may vary when using them – please report any issues to the wrapper first before reporting back to us.
```

Something like this?